docs/tutorials/HD143006_Part_2.py

```diff
@@ -0,0 +1,468 @@
+#
+# This tutorial will be going through how to initialize the model, the imaging and optimization process, how to improve the process through crossvalidation, and how to analyze the results of our work with Tensorboard.
+#
+# ### Loading Data
+# Let's load the data as we did in the previous HD143006 tutorial ([Part 1](HD143006_Part_1.html)) and create the MPoL Gridder object.
+#
+# *You can either download these two files (HD143006_continuum.fits and HD143006_continuum.npz) directly to your working directory, or use astropy to download them during run time.*
+
+# + cell_id="00001-be94721b-eee2-4e2e-96e2-dc65b9fd4f5b" deepnote_cell_type="code" deepnote_to_be_reexecuted=false execution_millis=663 execution_start=1623447169390 source_hash="4f0f20f8" tags=[]
+from astropy.io import fits
+import matplotlib.pyplot as plt
+import numpy as np
+from astropy.utils.data import download_file
+from torch.utils import tensorboard
+
+# downloading fits file
+fname_F = download_file(
+    "https://almascience.nrao.edu/almadata/lp/DSHARP/images/HD143006_continuum.fits",
+    cache=True,
+    pkgname="mpol",
+)
+# downloading extracted visibilities file
+fname_EV = download_file(
+    "https://zenodo.org/record/4904794/files/HD143006_continuum.npz",
+    cache=True,
+    pkgname="mpol",
+)
+# load extracted visibilities from npz file
+dnpz = np.load(fname_EV)
+uu = dnpz["uu"]
+vv = dnpz["vv"]
+weight = dnpz["weight"]
+data = dnpz["data"]
+# opening the fits file
+dfits = fits.open(fname_F)
+cdelt_scaling = dfits[0].header["CDELT1"] * 3600  # scaling [arcsec]
+cell_size = abs(cdelt_scaling)  # [arcsec]
+# close fits file
+dfits.close()
+
+# + cell_id="00008-d56e3afe-45cf-4fe5-a4e7-1803f28deec4" deepnote_cell_type="code" deepnote_to_be_reexecuted=false execution_millis=27 execution_start=1623441758279 source_hash="b76fed2d"
+from mpol import gridding, coordinates
+
+# creating Gridder object
+coords = coordinates.GridCoords(cell_size=cell_size, npix=512)
+gridder = gridding.Gridder(
+    coords=coords,
+    uu=uu,
+    vv=vv,
+    weight=weight,
+    data_re=data.real,  # separating the real and imaginary values of our data
+    data_im=data.imag,
+)
+
+# -
+
+# We now have everything from the last tutorial loaded and can begin the process of optimization and cross validation to improve our image quality.
+#
+# ### Getting the Dirty Image and Creating the Model
+#
+# First, we are going to get the dirty image from our gridder object. We will use the Briggs weighting scale and set `robust=0.0` here as these options lead to an already well optimized image (see [Part 1](HD143006_Part_1.html)).
```

Comment:
I'm not sure what you mean by the options lead to an already well-optimized image. The dirty image is just a possible starting point for the optimization, it's not the optimized RML image itself.